sound/soc/sof/ipc4-priv.h

```diff
@@ -28,11 +28,13 @@ enum sof_ipc4_mtrace_type {
 /**
  * struct sof_ipc4_fw_module - IPC4 module info
  * @sof_man4_module: Module info
+ * @fm_config: Pointer to the module config start of the module
  * @m_ida: Module instance identifier
  * @private: Module private data
  */
 struct sof_ipc4_fw_module {
 	struct sof_man4_module man4_module_entry;
+	const struct sof_man4_module_config *fm_config;
```

Comment:
what does 'fm' stand for?

Should this be config_start instead?

Reply:
renamed it to `fw_mod_cfg`, as it is coming from the firmware's manifest, bundled together.